.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,44 @@
+name: Node CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  test:
+    name: Node v${{ matrix.node-version }} on ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [8.x, 10.x, 12.x]
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Get yarn cache
+        id: yarn-cache
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - uses: actions/cache@v1
+        with:
+          path: ${{ steps.yarn-cache.outputs.dir }}
```

Comment:
why not just cache `node_modules` instead of the whole outputs dir

Reply:
I was playing around with cache and doing something like this works. 

![image](https://user-images.githubusercontent.com/18214059/69245834-8c257180-0be2-11ea-829c-ef69ad488572.png)

By the way, I just copy pasted that from this https://github.com/actions/cache/issues/94#issue-522791321

Reply:
I copied from https://github.com/actions/cache/blob/master/examples.md#node---yarn